kanban/api/views/issue.py

```diff
@@ -0,0 +1,67 @@
+from kanban.api.serializers import IssueSerializer
+from kanban.models.issue import Issue
+from kanban.tasks import (
+    issue_assignee_change_notification,
+    issue_due_time_notification,
+)
+from rest_framework import viewsets, generics
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from user_account.api.serializers import AssignUserSerializer
+from user_account.models.user_account import UserAccount
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+
+
+class IssueViewSet(viewsets.ModelViewSet):
+    queryset = Issue.objects.select_related("assignee", "project")
+    serializer_class = IssueSerializer
+
+    def perform_update(self, serializer):
+        previous_due_date = serializer.instance.due_date
+        instance = serializer.save()
```

Comment:
do you need save here? won't it be run automatically?

Reply:
Before save u have previous instance there, after save - new one so u can check if previous value is different than the new one.